### PR TITLE
Feature strating arhitecture

### PR DESCRIPTION
**Pull request by ethernal12**
I have created initial architecture, please for code review. @urosjarc

### CODE REVIEW

File: test/test1.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const assert = require("assert");
+const axios = require("axios").default;
+
+const req = axios.create({
+    baseURL: 'http://localhost:3000',
+    timeout: 1000
```
Comment (urosjarc) [inline test/test1.js]: O jebemti ma da mi daš na 10 sek, 1 sek je fullllllllllllllllllllll premal 💯

File: test/test1.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            })
+            .catch(function (error) {
+                // handle error
+                console.log(error);
+            })
+
+
+
+    })
+    it("Should have right JSON schema.", () => {
```
Comment (urosjarc) [inline test/test1.js]: Better description please.

**Review by urosjarc — CHANGES_REQUESTED** [review]
Please add changes to the code.

File: test/test1.js (hunk 2 above)
Comment (ethernal12) [reply test/test1.js]: Sem popravil šefe. @urosjarc

**Pull request merged**